docs/source/installing.rst

```diff
@@ -49,7 +49,8 @@ If you want to work on vaex for a Pull Request from the source, use the followin
 * make sure the dev versions of pcre are installed (e.g. ``conda install -c conda-forge pcre``)
 * install using (note: if you're on Windows, make sure that your command line/terminal has administrator privileges):
 
- * ``pip install -e .``  (again, use (ana)conda or virtualenv/venv)
+ * Create and activate a virtualenv/venv
+ * ``make init``
```

Comment:
If make doesn't work on windows, they won't be able to use the makefile at all :)
Do you have another good alternative? Because I don't want to also inline the `make deps`, that would be messy.

Reply:
Someone without `make` won't be able to use that file at all, and that's fine because it's a utility file.
But if they use the makefile they probably have `make`.
Do you have another solution for that? Because I don't want to change `make clean` for example, as it would be very messy.